sup3r/utilities/test_utils.py

```diff
@@ -182,6 +183,29 @@ def tke_spectrum(u, v):
     return E_k
 
 
+def tke_series(u, v):
+    """Longitudinal Turbulent Kinetic Energy Spectrum time series. Gives the
```

Comment:
longitudinal? Is that true?

Reply:
Yeah, it's being averaged over the latitude dimension

Reply:
ah i see that, interesting!